bundles/specmate-migration/src/com/specmate/migration/internal/services/MigratorService.java

```diff
@@ -92,6 +92,7 @@ public boolean needsMigration() throws SpecmateException {
 			// TODO using the exception for logic decisions is not good as we
 			// have more than 1 reason
 			// why an exception can occur.
+			System.out.println(e.getMessage());

```

Comment:
remove System.out.println, optionally replace with LogService call

Reply:
Should be fixed in PR 140.